src/java.base/share/classes/java/util/zip/ZipInputStream.java

```diff
@@ -172,13 +173,15 @@ public void closeEntry() throws IOException {
     }
 
     /**
-     * Returns 0 after EOF has reached for the current entry data,
-     * otherwise always return 1.
+     * Returns 0 when end of stream is detected for the current ZIP entry or
+     * {@link #closeEntry()} has been called on the current ZIP entry, otherwise
+     * return 1.
```

Comment:
Another nitpick: should this be `returns 1` to match `Returns 0` at line 176? Or vice versa?

Reply:
Sure updated